python/tests/plugins/test_help_dump_impl.py

```diff
@@ -37,6 +37,10 @@ def test_help_dump_prints_first_party_commands(self):
                 result.get_messages_as_string()
             )
 
+    def test_help_dump_json_dump_sort(self):
+        list_all_plugins = help_dump().messages
+        json.loads("".join(list_all_plugins), object_pairs_hook=OrderedDict)
+
```

Comment:
Here is a suggested change that asserts that command names in the help dump output are sorted alphabetically. 
```suggestion
        expected_commands = sorted(get_plugin_commands(), key=lambda command: command.name)

        list_all_plugins = help_dump().messages
        help_dump_output = json.loads("".join(list_all_plugins), object_pairs_hook=OrderedDict)

        expected_result = [command.name for command in expected_commands]
        actual_result = [command.get("name") for command in help_dump_output]

        self.assertListEqual(expected_result, actual_result)
```

Reply:
`expected_commands = sorted(get_plugin_commands()...` is throwing an unresolution error. I am testing locally for a fix

Reply:
Ok got it working locally...the test that is...now the test is failing so this may need to be looked into. There are differences between the expected and the actual results